src/FastAI.jl

```diff
@@ -24,10 +24,16 @@ using StaticArrays
 using Setfield
 using ShowCases
 using Tables
+using TextAnalysis
```

Comment:
Can you add the symbols used here as you did for WordTokenizers? Then it's easier to see where variables come from and the namespace isn't polluted